app/templates/bulletin/sunday/index.hbs

```diff
@@ -1,6 +1,55 @@
+<style>
+
+.header{
+  
+  text-align: center;
+
+}
+
+.anounce{
+
+list-style-type: none;
+
+}
+
+.pad{
+
+padding-bottom:20px;
+
+}
+
+</style>
+
+<div class='container'>
+<div class='row'><div class="col-sm-8 col-md-offset-2 header">
 <h1>{{name}}</h1>
+ <p>{{publishedAt}}</p>
+ <p class="lead">Welcome to Montreal Chinese Alliance Church's weekly English Sunday Worship Service.Please join us every week Sunday at 9:30 am. Our address is:<br> <b>
+13 Finchley, Hampstead, Quebec. </b></p>
+</div>
+</div> <!-- closes row -->
+
+<!-- place holder for image -->
+
+<img class="col-sm-8 col-md-offset-2 img-responsive pad" src='http://fc00.deviantart.net/fs71/f/2012/235/7/e/dzd_1080p_cod4_wallpaper_by_dzdigital-d5c3v89.jpg' alt="Sunday service Banner" />
+<div class="row">
+  <div class="col-sm-8 col-md-offset-2">   
+    <div class="col-sm-5">
+    <ul class="anounce">
+      <li>{{{serviceOrderHtml}}}</li>
+     </ul>
+     </div> <!-- end of sm-5 -->
+       <div class="col-sm-7">
+       	 <ul>
```

Comment:
You've got some hard tabs in the file... try to replace them with spaces. That's why the indentation looks all weird.